bot/commands/my100status.js

```diff
@@ -0,0 +1,51 @@
+const request = require('request');
+const authDetails = require('../../auth.json');
+const the100UserUrl = 'https://www.the100.io/api/v1/groups/3140/users?page=all';
+
+const my100status = msg =>
+{
+    const userName = msg.content.split(' ').pop();
+
+    const lcUserName = userName.toLowerCase();
+
+    const userAuthOptions =
+    {
+		url: the100UserUrl,
+        headers:
+        {
+			'Authorization': authDetails.the100token
+        }
+    };
+
+    const getUserStatus = (error, response, body) =>
+    {
+        if (!error && response.statusCode == 200)
+        {
+            const usersJson = JSON.parse(body);
+
+            const requestedUser = usersJson.filter(gamer =>
+            {
+				return gamer.gamertag.toLowerCase() == lcUserName;
+			});
+
+            const gamer = requestedUser[0];
+
+            if (gamer === undefined)
+            {
+                msg.reply('Sorry, no user was found by that name. Please try again with a different username.');
+			}
+            else
+            {
+                msg.reply(gamer.gamertag + ' has ' + gamer.activity_score + ' activity points and ' + gamer.karmas_count + ' karma on the site!');
+			}
+		}
+        else
+        {
```

Comment:
This else does nothing and indentation is jacked. We should either have an error handler to handle not getting 200 from the server or just remove the else all together prob.

Reply:
This was also used during testing and the else statement has been removed.